README.md

```diff
@@ -27,9 +27,17 @@ Eiffel REMReM Generate takes a partial message in JSON format, validates it and
 
 # Documentation
 
-1. [**Running RemRem-Generate service**]()
+1. [**Introduction**](wiki/markdown/index.md)
+
+2. [**Pre-requisites**](wiki/markdown/index.md#Pre-requisites)
+
+3. **Running REMReM-Generate service**
+    - [**Installation**](https://eiffel-community.github.io/eiffel-remrem-generate/#installation)
```

Comment:
Why not point to the markdown file inside the repository? Isn't there one?

Reply:
In installation page , currently getting latest wars file by using javascript code. Unable to add java script code  in md file. So, as of now we are redirecting installation page to gh-pages.

Reply:
Ok, I don't really see the need to having that javascript code in the first place. Isn't it enough with a link to Jitpack? There users can download remrem themselves (and select any version). We also have a JitPack label at the top of README...

Reply:
Hi Emelie,

Updated the suggested changes.